include/seqan3/alphabet/gap/gap.hpp

```diff
@@ -70,8 +72,16 @@ struct gap
     using rank_type = bool;
 
     /* member */
-    //! internal value
+    //! \privatesection
     static constexpr rank_type value = 0;
+    //! \publicsection
+
+    /*! @name letter values
+     * Static member "letters" that can be assigned to the alphabet or used in aggregate initialization.
+     */
+    //!@{
```

Comment:
this is a group of letters as you also use in dna4.

For the sake of consistency in the generated doc, I would keep this group, even though it has only one member.